src/feature/dirauth/process_descs.c

```diff
@@ -428,15 +428,20 @@ dirserv_free_fingerprint_list(void)
 
 /** Return -1 if <b>ri</b> has a private or otherwise bad address,
  * unless we're configured to not care. Return 0 if all ok. */
-static int
+int
```

Comment:
We use the "STATIC" macro for functions that are static in binaries, but used in the unit tests:
```suggestion
STATIC int
```